translations/ru/get5.phrases.txt

```diff
@@ -6,31 +6,47 @@
     }
     "WaitingForCastersReadyInfoMessage"
     {
-        "ru"            "Ожидаем, пока все игроки напишут {2} для начала."
+        "ru"            "Ожидаем, пока {1} напишут {2} для начала."
     }
     "ReadyToRestoreBackupInfoMessage"
     {
-        "ru"            "Напиши {1} когда твоя команда будет готова загрузить сохраненную игру."
+        "ru"            "Напишите {1} когда вы будете готовы загрузить сохраненную игру."
     }
     "ReadyToKnifeInfoMessage"
     {
-        "ru"            "Напиши {1} когда твоя команда будет готова к ножевому раунду."
+        "ru"            "Напишите {1} когда вы будете готовы к ножевому раунду."
     }
     "ReadyToStartInfoMessage"
     {
-        "ru"            "Напиши {1} когда твоя команда будет готова начать."
+        "ru"            "Напишите {1} когда вы будете готовы начать."
+    }
+    "YouAreReady"
+    {
+        "ru"            "Вы были отмечены как готовые."
+    }
+    "YouAreReadyAuto"
+    {
+        "ru"            "ПРИМЕЧАНИЕ: Вы были отмечены как готовые из-за игровой активности. Введите {1}, если вы не готовы."
+    }
+    "YouAreNotReady"
+    {
+        "ru"            "Вы были отмечены как НЕ готовые."
     }
     "WaitingForEnemySwapInfoMessage"
     {
-        "ru"            "{1} выиграла раунд. Ожидаем, пока они напишут {2} или {3}."
+        "ru"            "{1} выиграли раунд. Ожидаем, пока они напишут {2} или {3}."
     }
     "WaitingForGOTVBrodcastEndingInfoMessage"
     {
         "ru"            "Карта будет изменена, когда закончится трансляция на GOTV."
     }
+    "WaitingForGOTVVetoInfoMessage"
+    {
+        "ru"            "Карта изменится, как только на GOTV будут показаны вето на карты."
+    }
     "NoMatchSetupInfoMessage"
     {
-        "ru"            "На этом сервере нет матча."
+        "ru"            "На данном сервере не настроен матч."
```

Comment:
Remove the full stop. The game adds it automatically to phrases with type `KickedNote`: https://splewis.github.io/get5/dev/translations/#kickednote